MapboxNavigation/Resources/ca.lproj/Abbreviations.csv.plist

```diff
@@ -3,69 +3,121 @@
 <plist version="1.0">
   <dict>
     <key>abbreviations</key>
-    <string>&quot;creek&quot;, &quot;Crk&quot;
-&quot;center&quot;, &quot;Ctr&quot;
-&quot;national&quot;, &quot;Nat’l&quot;
-&quot;mount&quot;, &quot;Mt&quot;
-&quot;mountain&quot;, &quot;Mtn&quot;
-&quot;crossing&quot;, &quot;Xing&quot;
-&quot;downtown&quot;, &quot;Dtwn&quot;
-&quot;international&quot;, &quot;Int’l&quot;
-&quot;park&quot;, &quot;Pk&quot;
-&quot;square&quot;, &quot;Sq&quot;
-&quot;saints&quot;, &quot;SS&quot;
-&quot;heights&quot;, &quot;Hts&quot;
-&quot;route&quot;, &quot;Rte&quot;
-&quot;saint&quot;, &quot;St&quot;
-&quot;fort&quot;, &quot;Ft&quot;
-&quot;market&quot;, &quot;Mkt&quot;
-&quot;centre&quot;, &quot;Ctr&quot;
-&quot;william&quot;, &quot;Wm&quot;
-&quot;school&quot;, &quot;Sch&quot;
-&quot;senior&quot;, &quot;Sr&quot;
-&quot;river&quot;, &quot;Riv&quot;
-&quot;sister&quot;, &quot;Sis&quot;
-&quot;village&quot;, &quot;Vil&quot;
-&quot;station&quot;, &quot;Sta&quot;
-&quot;apartments&quot;, &quot;apts&quot;
-&quot;university&quot;, &quot;Univ&quot;
-&quot;township&quot;, &quot;Twp&quot;
-&quot;lake&quot;, &quot;Lk&quot;
-&quot;junior&quot;, &quot;Jr&quot;
-&quot;father&quot;, &quot;Fr&quot;
-&quot;point&quot;, &quot;Pt&quot;
-&quot;memorial&quot;, &quot;Mem&quot;
-&quot;junction&quot;, &quot;Jct&quot;
+    <string>&quot;Cim&quot;,&quot;cim&quot;
+&quot;Coll&quot;,&quot;coll&quot;
+&quot;Comunicacions&quot;,&quot;com.&quot;
+&quot;Xarxa Hidrogràfica&quot;,&quot;curs fluv.&quot;
+&quot;Edificació&quot;,&quot;edif.&quot;
+&quot;Equipament&quot;,&quot;equip.&quot;
+&quot;Element Hidrogràfic&quot;,&quot;hidr.&quot;
+&quot;Indret&quot;,&quot;indr.&quot;
+&quot;Relleu del Litoral&quot;,&quot;lit.&quot;
+&quot;Municipi&quot;,&quot;mun.&quot;
+&quot;Orografia&quot;,&quot;orogr.&quot;
+&quot;Massís&quot;,&quot; Serra&quot;,&quot;serra&quot;
```

Comment:
We don't support synonyms like this, do we? I think we need to split this line up in the translated string in Transifex.

Reply:
Fixed in Transifex. However, this is only the csv version of the plist so the English fallback will be used. Will fix in a follow up PR.

Reply:
Fixed in #750